### DISCUSSION TITLE
get annoted content and bbox

### DISCUSSION THREAD

**Discussion by hehuang139** [jsvine/pdfplumber#749] [Q&A]
can i get annoted content and bbox? not anntations content.

i can't find any way to get annoted content . i can only get annotations attribute by page.annots

**Comment by jsvine** [comment]
The annotated content is, as far as I'm aware, not typically stored alongside the annotations data. However, depending on the particular annotation type, you may be able to get what you need via `Page.crop((x0, top, x1, bottom))` with values corresponding to `my_annot["x0"]`/etc., followed by `cropped_page.extract_text(...)`.
